xcm/pallet-xcm/src/lib.rs

```diff
@@ -1205,6 +1208,7 @@ impl<T: Config> Pallet<T> {
 		]);
 		let mut message = Xcm(vec![
 			WithdrawAsset(assets),
+			SetFeesMode { jit_withdraw: true },
```

Comment:
Is this currently correct? The InitiateTeleport changed in master but we also added SetFeesMode.

Reply:
Yes, this is fine.